tests/jax/ops/test_negation.py

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import jax.numpy as jnp
+import pytest
+from infra import run_op_test_with_random_inputs
+
+
+@pytest.mark.parametrize("x_shape", [(32, 32), (64, 64)])
+def test_negation(x_shape: tuple):
+    def negation(x: jax.Array) -> jax.Array:
+        return jnp.negative(x)
+
+    # Try both negative and positive values.
```

Comment:
Change to "Trying" instead of "Try", this way it seems like a TODO comment :) Same for `tests/jax/ops/test_sign.py`